src/ibis_birdbrain/bot.py

```diff
@@ -89,12 +90,20 @@ def __init__(
         body = """TODO"""  # noqa
 
         # system initialization message
+        attachments = [
+                DatabaseAttachment(con, description=self.data_description),
+        ]
+        # need a better way to handle this
+        if str(cached_table) != "None":
```

Comment:
Nit: Do we need to convert it to str? Unless the `cached_table` can return `"None"` as a string -- we should be just able to do `if cached_table is not None`, right?

Reply:
You are right. I was using `Basebackend`, it did not work for `None`. Thanks a lot.